railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -62,7 +62,7 @@ Rails.application.configure do
   # Don't log any deprecations.
   config.active_support.report_deprecations = false
 
-  # Replace the default in-process memory cache store with a durable alternative.
+  # Replace the default file cache store with a durable alternative.
   # config.cache_store = :mem_cache_store
```

Comment:
Here `mem_cache_store` refers to `memcached`.

The comment is indeed inacurate, but because there is no default.

Reply:
Thank you for the review. Indeed, [lookup_store](https://github.com/rails/rails/blob/2a0e057570cbfd39fc38e35d97d1c43112c44ebf/activesupport/lib/active_support/cache.rb#L94-L95) defaults to MemoryStore. 💪